build.gradle

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2014 Netflix, Inc.
+ * Copyright (c) 2018, salesforce.com, inc. All rights reserved.
```

Comment:
Is this a requirement from your legal department? If it's not, the convention is to leave the headers pointing at the original authors.

Reply:
@lwander yes, it's a requirement from the legal department.

Reply:
Typically a CLA is used to transfer the copyright but it doesn't appear that Spinnaker has a CLA.  @lwander can you confirm?

Reply:
Yes that's right, there is no CLA at this time